sys/kern/main.c

```diff
@@ -99,8 +102,8 @@ __noreturn void kernel_init(void) {
   /* Mount filesystems (including devfs). */
   mount_fs();
 
-  /* First (FTTB also last) stage of device init. */
-  init_devices();
+  /* First stage of device init. */
+  init_devices(FIRST_PASS);
```

Comment:
It'd be great if you described what kind of drivers should be initialized by `FIRST_PASS`. Our goal was to start those that do not require `devfs`, but provide first timer and console output.

Reply:
Done.